tests/test_data.py

```diff
@@ -871,3 +871,46 @@ def test_dnode_new_opaq_find_one(self):
         opaq_hostname = dnode.find_one("hostname")
 
         self.assertIsInstance(opaq_hostname, DLeaf)
+
+    def test_attr(self):
+        dnode = self.ctx.parse_data_mem(self.JSON_CONFIG, "json", validate_present=True)
+
+        hostname = dnode.find_one("hostname")
+        hostname.free(with_siblings=False)
+
```

Comment:
Same question as the other commit, can you explain why you load the configuration, and delete the hostname in it at the beginning of your test? Why not start with an empty configuration instead?

Reply:
Done for all the tests